### PR TITLE
Format code with goimports

### PR DESCRIPTION
**Pull request by muesli**
Fixed formatting and import order.

### CODE REVIEW

File: suite/suite_test.go [hunk 1]
```diff
@@ -350,7 +350,7 @@ func TestRunSuite(t *testing.T) {
 type SuiteSetupSkipTester struct {
 	Suite
 
-	setUp bool
+	setUp    bool
```
Comment (xorcare) [inline suite/suite_test.go]: these changes are not related to imports

File: suite/suite.go [hunk 2]
```diff
@@ -84,7 +84,7 @@ func Run(t *testing.T, suite TestingSuite) {
 	defer failOnPanic(t)
 
 	suiteSetupDone := false
-	
+
```
Comment (xorcare) [inline suite/suite.go]: these changes are not related to imports

File: suite/suite_test.go (hunk 1 above)
Comment (muesli) [reply suite/suite_test.go]: goimports does both import ordering as well as calling gofmt.

Comment (bencampbell-wf) [reply suite/suite_test.go]: Hmm. This change would lead me to believe the go formatting check (defined during CI)[https://github.com/stretchr/testify/blob/master/.travis.gofmt.sh#L5) is not being run.

FYI: @georgelesica-wf

Comment (georgelesica-wf) [reply suite/suite_test.go]: Agreed, weird...

Comment (georgelesica-wf) [reply suite/suite_test.go]: Fixed, hopefully... https://github.com/stretchr/testify/pull/804
